lib/promises-aplus/promises-aplus-test-adapter.js

```diff
@@ -11,7 +11,8 @@ var minErr = function minErr (module, constructor) {
   };
 };
 
-var $q = qFactory(process.nextTick, function noopExceptionHandler() {});
+var noop = function() {};
+var $q = qFactory(noop, process.nextTick, noop);
```

Comment:
this doesn't look right. qFactory signature is `function qFactory(nextTick, exceptionHandler) {` no?